### PR TITLE
Raise $wgRCMaxAge to 720 days; convert it to days (T6417)

### PR DESCRIPTION
**Pull request by Universal-Omega**
`UPDATE mw_settings SET s_settings = JSON_REPLACE(s_settings, '$.wgRCMaxAge', CEIL(s_settings->>'$.wgRCMaxAge' / 86400))`
then, `rebuildrecentchanges.php`
will need run if this is merged

### CODE REVIEW

**Comment by paladox** [comment]
Any reason why we need to do this? This would have to be done on over 3,000 wikis.

**Comment by Universal-Omega** [comment]
> Any reason why we need to do this? This would have to be done on over 3,000 wikis.

1) Much much easier to know what to set the value to
2) It would make the numbers smaller so they actually fit in the integer box.

File: LocalWiki.php [hunk 1]
```diff
@@ -98,6 +98,9 @@ function onInactiveSiteNoticeAfter( &$siteNotice, $skin ) {
  	];
 }
 
+// $wgRCMaxAge
+$wgRCMaxAge = ceil($wgRCMaxAge/86400);  // Convert to seconds
```
Comment (paladox) [inline LocalWiki.php:102]: Won't work. ManageWiki would override.

Comment (Universal-Omega) [reply LocalWiki.php:102]: oh, so would `$wgRCMaxAge = ceil($wmgRCMaxAge/86400);` and make the managewiki one, wmg?

Comment (paladox) [reply LocalWiki.php:102]: yeh.

Comment (Universal-Omega) [reply LocalWiki.php]: done

**Comment by BlackWidowMovie0** [comment]
@paladox Anything I can help with over here? If so, just point me in the right direction 👍🏾 . If not, that's ok, I guess 👎🏾.

**Comment by Universal-Omega** [comment]
Will just use seconds

**Pull request closed without merging**